org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/services/format/wtp/TestPartitionFormatterXML.java

```diff
@@ -0,0 +1,342 @@
+package org.eclipse.lemminx.services.format.wtp;
+
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import org.eclipse.lemminx.services.XMLLanguageService;
+import org.eclipse.lemminx.settings.SharedSettings;
+import org.eclipse.lemminx.settings.XMLFormattingOptions;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test from
+ * https://github.com/kingargyle/wtp-sourceediting/blob/master/tests/org.eclipse.wst.xml.core.tests/src/org/eclipse/wst/xml/core/tests/format/TestPartitionFormatterXML.java

```

Comment:
You should keep copyright headers from original sources

Reply:
fixed